mypostcard/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"bytes"
+	"embed"
+	"flag"
+	"fmt"
+	"image"
+	"image/draw"
+	"image/png"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/golang/freetype/truetype"
+	"golang.org/x/image/font"
+	"golang.org/x/image/math/fixed"
+)
+
+//go:embed LICENSE
```

Comment:
Maybe put in a single block and document nicely?

Reply:
:) it was at first - but wanted to copy-paste specific lines from the code to the blog